srv/admin-service.cds

```diff
@@ -1,34 +1,43 @@
 using {sap.common.Languages as CommonLanguages} from '@sap/cds/common';
 using {my.bookshop as my} from '../db/index';
 using {sap.changelog as changelog} from 'com.sap.cds/change-tracking';
-using {sap.attachments.Attachments} from `com.sap.cds/cds-feature-attachments`;
+using {my.common.Hierarchy as Hierarchy} from './hierarchy';
+using {sap.attachments.Attachments} from`com.sap.cds/cds-feature-attachments`;
 
 extend my.Orders with changelog.changeTracked;
 
-@path : 'admin'
-service AdminService @(requires : 'admin') {
-  entity Books   as projection on my.Books excluding { reviews } actions {
-    action addToOrder(order_ID : UUID, quantity : Integer) returns Orders;
-  }
+@path: 'admin'
```

Comment:
there are many unrelated formatting changes in this file. Please minimized the diff with `main`